.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,11 @@
-<!-- Thank you for creating a pull request!
+<!-- Thank you for creating a pull request! -->
 
-Please go through [our contribution guide][CONTRIBUTING.md] to hopefully get your changes merged quicker.
+##### Checklist before submitting
 
-[CONTRIBUTING.md]: https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md -->
+- [ ] Did you read our [contributor guideline](https://github.com/optuna/optuna/blob/master/CONTRIBUTING.md), especially "Creating a Pull Request" section?
```

Comment:
A checkbox to read the guide makes perfect sense, however, it might be somewhat demanding to a new contributor, manifesting is like a "contract". This is not a strong opinion, but adding the headers below might suffice as a first step?

Reply:
I reverted these lines as comments. Is it fine?